packages/client/src/runtime/core/types/Extensions.ts

```diff
@@ -26,7 +26,7 @@ export type Args = InternalArgs
 export type DefaultArgs = InternalArgs<{}, {}, {}, {}>
 
 export type GetResult<Base extends Record<any, any>, R extends Args['result'][string], KR extends keyof R = string extends keyof R ? never : keyof R> =
-  { [K in KR | keyof Base]: K extends KR ? R[K] extends (() => { compute: (...args: any) => infer C }) ? C : never : Base[K] }
+  { [K in KR | keyof Base]: K extends KR ? R[K] extends (() => { compute: (...args: any) => infer C }) ? C : never : Base[K] } & unknown
```

Comment:
>so I left a comment in the code.
Did you push the comment? 😄

Reply:
Nope :homer-simpson-disappears-into-bush:

Reply:
https://github.com/prisma/prisma/pull/20319/files#diff-17bc9f33a3484a25fedb3eaaeb439e7cd240ec835fc020953241d0c2b075c3e5R30